pages/common/git-rev-list.md

```diff
@@ -0,0 +1,16 @@
+# git rev-list
+
+> Lists commits in reverse chronological order.
```

Comment:
This description may make it seem like the "rev" in the command name stands for "reverse". I'd suggest incorporating "revision" in the sentence, to make sure we avoid conveying that misconception.

Reply:
Good point. See 3dc463b5a70d58485f5e7a87387563bc9302e854